src/basic/missing.h

```diff
@@ -1351,4 +1351,11 @@ struct fib_rule_uid_range {
 #define NS_GET_NSTYPE _IO(0xb7, 0x3)
 #endif
 
+#if ! HAVE_STRUCT_FIB_RULE_UID_RANGE
+struct fib_rule_uid_range {
+        __u32 start;
+        __u32 end;
+};
+#endif
```

Comment:
the exact same definition already exists a few lines earlier?

Reply:
Yeah, sorry, didn't notice.  That patch will be skipped.